apps/docs/content/guides/platform/manage-your-usage/read-replicas.mdx

```diff
@@ -0,0 +1,75 @@
+
+**Provisioned Disk IOPS (optional)**
+Read Replicas inherit any additional provisioned Disk IOPS from the primary database. Refer to [Manage Disk IOPS usage]() for details on how charges are calculated.
+
+**Provisioned Disk Throughput (optional)**
+Read Replicas inherit any additional provisioned Disk Throughput from the primary database. Refer to [Manage Disk Throughput usage]() for details on how charges are calculated.
+
+**IPv4 (optional)**
+If the primary database has a configured IPv4 address, its Read Replicas are also assigned one, with charges for each. Refer to [Manage IPv4 usage]() for details on how charges are calculated.
+
+### Read Replica usage on your invoice
+Compute incurred by Read Replicas is shown as "Replica Compute Hours" on your invoice. Disk Size, Disk IOPS, Disk Throughput and IPv4 are not shown separately for Read Replicas.
+
+## Billing examples
+### No additional resources configured
+The project has no IPv4 and no additional Disk IOPS and Disk Throughput configured.
+
+| Line Item                     | Units     | Costs      |
+|-------------------------------|-----------|------------|
+| Pro Plan                      | 1         | $25        |
+|                               |           |            |
+| Compute Hours Small Project 1 | 744 hours | $15        |
+| Disk Size Project 1           | 8 GB      | $0         |
+|                               |           |            |
+| Compute Hours Small Replica   | 744 hours | $15        |
+| Disk Size Replica             | 10 GB     | $1.27      |
+|                               |           |            |
+| **Subtotal**                  |           | **$56.27** |
+| Compute Credits               |           | -$10       |
+| **Total**                     |           | **$46.27** |
+
+### Additional resources configured
+The project has IPv4 and additional Disk IOPS and Disk Throughput configured.
+
+| Line Item                     | Units     | Costs       |
+|-------------------------------|-----------|-------------|
+| Pro Plan                      | 1         | $25         |
+|                               |           |             |
+| Compute Hours Large Project 1 | 744 hours | $110        |
+| Disk Size Project 1           | 8 GB      | $0          |
+| Disk IOPS Project 1           | 3600      | $14.40      |
+| Disk Throughput Project 1     | 200 MiB/s | $7.13       |
+| IPv4 Project 1                | 1         | $4          |
+|                               |           |             |
+| Compute Hours Large Replica   | 744 hours | $110        |
+| Disk Size Replica             | 10 GB     | $1.27       |
+| Disk IOPS Replica             | 3600      | $14.40      |
+| Disk Throughput Replica       | 200 MiB/s | $7.13       |
+| IPv4 Replica                  | 1         | $4          |
```

Comment:
Maybe add a second replica in this example?

Reply:
Added a second one